zerver/data_import/gitter.py

```diff
@@ -338,6 +338,7 @@ def do_convert_data(gitter_data_file: str, output_dir: str, threads: int = 6) ->
     create_converted_data_files([], output_dir, "/uploads/records.json")
     # IO attachments records
     create_converted_data_files(attachment, output_dir, "/attachment.json")
+    create_converted_data_files({"service": "slack"}, output_dir, "/meta.json")
```

Comment:
What is `meta.json`?  Oh, is this just generating a tiny file with just the `server => slack` field in it?  

I would prefer to store this sort of detail in a field in `realm.json` that isn't imported (and perhaps we should have `manage.py export` set it as well).